packages/usa-input-mask/src/usa-input-mask.twig

```diff
@@ -9,9 +9,18 @@
       placeholder="{{ placeholder }}" 
       pattern="{{ pattern }}" 
       class="usa-input usa-masked" 
-      aria-describedby="{{ id }}Hint"
+      aria-describedby="{{ id }}Error"
```

Comment:
I'm unsure if this change is more/less or just as accessible as it was before. I'll look further into this when testing screen readers.